src/config.rs

```diff
@@ -38,6 +39,7 @@ impl Device {
             disksize: 0,
             swap_priority: 100,
             mount_point: None,
+            mount_options: Some("".to_string()),
```

Comment:
This makes no sense. Either make this just String, or actually only write it if it's set.

Reply:
@nabijaczleweli sorry that I have zero experience on rust, and what I could handle is modify from similar logic, e.g. `fs-type` and `mount-point`.

What am I looking for from here is: if `mount-options` is empty string or None, just print an empty string to the generated file with `Options=""`.

May you provide some code snippet as reference, so I could modify it correctly?